test_unstructured_ingest/test-ingest.sh

```diff
@@ -12,17 +12,15 @@ scripts=(
 'test-ingest-s3.sh'
 'test-ingest-s3-minio.sh'
 'test-ingest-azure.sh'
-'test-ingest-biomed-api.sh'
-'test-ingest-biomed-path.sh'
-## NOTE(yuming): The following test should be put after any tests with --preserve-downloads option
-'test-ingest-pdf-fast-reprocess.sh'
 'test-ingest-box.sh'
 'test-ingest-discord.sh'
 'test-ingest-dropbox.sh'
 'test-ingest-github.sh'
 'test-ingest-gitlab.sh'
 'test-ingest-google-drive.sh'
 'test-ingest-wikipedia.sh'
+'test-ingest-biomed-api.sh'
+'test-ingest-biomed-path.sh'
```

Comment:
is this diff needed?

Reply:
This probably comes from when I pull and merge with main. I will edit that!